lib/broccoli/ember-app.js

```diff
@@ -818,8 +818,8 @@ EmberApp.prototype.appAndDependencies = function() {
   });
 
   var postprocessedApp = this.addonPostprocessTree('js', preprocessedApp);
+
   var sourceTrees = [
-    external,
```

Comment:
Needs review: I'm assuming here that the external tree isn't going to provide anything that matches the regexp used by `_es6ifyAppModules` below.